### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by midolight**
Create open source file and add documentation

### CODE REVIEW

File: opensource.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Hello Open Source World
+Add Documentation to this project
```
Comment (mehdiwadoud) [inline opensource.txt]: you need to add specifications' lines

**Review by mehdiwadoud — CHANGES_REQUESTED** [review]
review completed

File: opensource.txt (hunk 1 above)
Comment (midolight) [reply opensource.txt]: okey as soon as possible

File: opensource.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+Add Specifications' lines as requested by the owner
```
Comment (mehdiwadoud) [inline opensource.txt:3]: good job !!

**Review by mehdiwadoud — APPROVED** [review]
good job

**Pull request merged**